source/common/stats/stats_impl.h

```diff
@@ -23,6 +24,9 @@
 
 #include "absl/strings/string_view.h"
 
+extern "C" {
```

Comment:
cc @jmillikin-stripe we might be able to just compile the library as C++ to avoid this. Not sure.

Reply:
circllhist.h ought to have C/C++ compatibility guards.

```
#ifdef __cplusplus
extern "C" {
#endif  // __cplusplus

...

#ifdef __cplusplus
}
#endif  // __cplusplus
```